core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7441,6 +7441,9 @@ components:
         message:
           description: An optional transaction message. Only affects the costing.
           $ref: "#/components/schemas/TransactionMessage"
+        opt_ins:
```

Comment:
Sorry for confusion, on Core API, it looks like we call these "Options" see e.g.:

```
sbor_format_options:
          $ref: "#/components/schemas/SborFormatOptions"
```

So we should probably call this:
```
response_options:
    $ref: "#/components/schemas/TransactionPreviewResponseOptions"
```

Reply:
Ah, options sounds good to me! I will change the name!